### DISCUSSION TITLE
ESM for browser doesn't work

### DISCUSSION THREAD

**Discussion by manake** [josdejong/mathjs#2784] [Q&A]
When I try to `import * as mathjs from ./src/index.js` I get `Uncaught TypeError: Failed to resolve module specifier "typed-function". Relative references must start with either "/", "./", or "../".`

This is after `npm run build`.

It would be nice if /src/ was ESM version importable into browser without any prior building required.

**Comment by josdejong** [comment]
Why aren't you using the npm package? Just install it via `npm install mathjs`.
